tests/bootstrap.php

```diff
@@ -1,17 +1,19 @@
 <?php
 
 define("MOCK_MINIMUM_VERSION", "0.40.0");
+define("MOCK_DOMAIN", getenv("STRIPE_MOCK_DOMAIN") ?: 'localhost');
 define("MOCK_PORT", getenv("STRIPE_MOCK_PORT") ?: 12111);
 
 // Send a request to stripe-mock
-$ch = curl_init("http://localhost:" . MOCK_PORT . "/");
+$mock_url = MOCK_DOMAIN . ":" . MOCK_PORT . "/";
```

Comment:
Add the protocol here too:
```suggestion
$mock_url = "http://" . MOCK_DOMAIN . ":" . MOCK_PORT . "/";
```